buzz_bot.py

```diff
@@ -0,0 +1,43 @@
+"""Main file for the discord bot"""
+import json
+import discord
+import counting_logic as count
+import update_users
+
+# auth token from auth.json
+token = json.load(open('auth.json', 'r'))['token']
+
+
+class MyClient(discord.Client):
+    """Our main discord bot class"""
+    async def on_ready(self):
+        """Stuff that happens when the bot is turned on"""
+        print("buzz-bot is on")
+        await update_users.first_time(self)
+
+    async def on_member_join(self, member):
+        """Stuff that happens when there is a new member"""
+        await update_users.new_member(member)
+
+    async def on_member_remove(self, member):
+        """Stuff that happens when a member leaves"""
```

Comment:
I think these comments aren't very necessary, the names of the methods are good enough